### PR TITLE
Restrict access to SQL explorer to superusers only

### PR DESCRIPTION
**Pull request by jamienoss**
Resolves [Notion ticket](https://www.notion.so/Restrict-SQL-explorer-access-to-superusers-only-d46e911f4b8543e2bd6bdb20b1dc315b?pvs=4)

### CODE REVIEW

**Comment by jamienoss** [comment]
docker image requires git...

File: biospecdb/urls.py [hunk 1]
```diff
@@ -13,12 +13,16 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+
+from decorator_include import decorator_include
```
[the remark this answers is not in the document]
Comment (jamienoss) [reply biospecdb/urls.py:17]: > Or it's a function with the same name as a module?

Correct.

**Review by vkluzner — APPROVED** [review]
Looks good now - I approved it.
Just a last comment - do you want to add a remark about the installation of `django-decorator-include` in the README file?

**Comment by jamienoss** [comment]
Thanks for the review.

>do you want to add a remark about the installation of django-decorator-include in the README file

No, we don't for any others.

**Pull request merged**